model_utilities/train/finetuner.py

```diff
@@ -42,53 +44,68 @@ def __plot_losses(self, training_losses: list[float], validation_losses: list[fl
         plt.ylabel("Loss")
         plt.legend()
         plt.savefig("./debug/graphs/losses.png")
-
-    def __train(self, model: AutoModelForSeq2SeqLM, tokenizer: NllbTokenizer, dataset: ds.Dataset, optimizer: Adafactor,
+        
+    def __tokenize(self, dataset: ds.Dataset, src_lang: str, tgt_lang: str, tokenizer: NllbTokenizerFast, max_length : int) -> BatchEncoding:
+        tokenizer.src_lang = src_lang
+        tokenizer.tgt_lang = tgt_lang
+        tokenized_input = tokenizer(dataset[src_lang], text_target=dataset[tgt_lang], return_tensors='pt', padding=True, truncation=True, max_length=max_length)
```

Comment:
```suggestion
        tokenized_input = tokenizer(dataset[src_lang], text_target=dataset[tgt_lang], return_tensors="pt", padding=True, truncation=True, max_length=max_length)
```
Let's keep the uniform quoting style across the codebase.